### PR TITLE
Rules Engine - Edge Firewall_EN

### PR DESCRIPTION
**Pull request by Luize-boyen**
Doc improvement and corrections

### CODE REVIEW

File: edge-firewall/rules-engine/index.md [hunk 1]
```diff
@@ -2,24 +2,35 @@
...
+The *Rules Engine* is designed for you to be able to implement your security logic at the edge. Based on programmable criteria, you choose what you want to do.
 
-> 1. *[How does it work?](#how-does-it-work)*
-> 2. *[Rules](#rules)*
-> 3. *[Hands-on. Step by step to configure Azion Edge Firewall](#hands-on-step-by-step-to-configure-azion-edge-firewall)*
-> 4. *[Hands-on. Creating a configuration with the Web Application Firewall Module using Criteria and Behavior](#hands-on-waf-criteria-behavior)*
-> 5. *[Hands-on. Creating a Web Application Firewall configuration using the WAF Rule Set for your application](#hands-on-waf-rule-sets)*
+#### With *Rules Engine* you can:
+
+- Block;
```
Comment (rodrigocst) [inline edge-firewall/rules-engine/index.md:9]:
```suggestion
- Block the request;
```

File: edge-firewall/rules-engine/index.md [hunk 2]
```diff
@@ -136,98 +150,97 @@ In Behavior you add the behaviors you want to perform, if the rule's conditions
...
 
 **Behavior**:
 
-- Then: Choose Set Rate Limit
+- **Then**: Choose *Set Rate Limit*
 
-    e.g.: *Then* *Set Rate Limit*, *Average Rate* *Limit* *(req/s) 2*, *Client IP Address*, *Maximum burst size 10*. In this example, for the Set Rate Limit action, the additional fields Average Rate Limit (req/s) and Maximum Burst Size will appear, in addition to the restriction selection option by IP or Global address, that is, if the request meets the criteria of the Rule, then a rate limit of 2 requests per second, per IP address will be set, with a burst limit of 10 requests.
+    Example.: *Then* *Set Rate Limit*, *Average Rate* *Limit* *(req/s) 2*, *Client IP Address*, *Maximum burst size 10*. In this example, for the Set Rate Limit action, the additional fields Average Rate Limit (req/s) and Maximum Burst Size will appear, in addition to the restriction selection option by IP or Global address, that is, if the request meets the criteria of the Rule, then a rate limit of 2 requests per second, per IP address will be set, with a burst limit of 10 requests.
```
Comment (rodrigocst) [inline edge-firewall/rules-engine/index.md:193]: Hi Luize!
Here we add the feature of rate-limiting / minute! I suggest you cover this information also at this step.
Thanks!

**Review by rodrigocst — CHANGES_REQUESTED** [review]

File: edge-firewall/rules-engine/index.md (hunk 2 above)
Comment (Luize-boyen) [reply edge-firewall/rules-engine/index.md:193]: Hi, Rodrigo!
Thanks! I will change it! ;)

File: edge-firewall/rules-engine/index.md (hunk 1 above)
Comment (Luize-boyen) [reply edge-firewall/rules-engine/index.md]: Arrumado! :)

**Review by VitorEltz-Azion — APPROVED** [review]

**Pull request merged**